### ISSUE TITLE
A11y: Voice over doesn't read discount amount as negative on HTML document

### ISSUE THREAD

**Issue by shopwareBot** [shopware/platform#7265]
*Note*: Issue happens on 6.6 and 6.7

Can reproduce with attached html instead of following below steps: [^Invoice 1006.html]

*Steps*:

# Go to Admin → Settings → Documents
# Open invoice
# Turn on *Display document in "My account"*
# In Storefront, login as customer and create an order with promotion
# Go back to Admin → Orders → Create an document → Create invoice and send the document
# Customer open the order and view created HTML document, for example: [^Invoice 1006.html] 
# Open Voice Over and navigate to discount cell

*Actual*: Voice Over read the number without minus sign, see attached video

!image-20250304-143658.png|width=962,height=737,alt="image-20250304-143658.png"!

!Understanding Invoice Details 📄.mp4|width=1920,height=1080,alt="Understanding Invoice Details 📄.mp4"!

**Comment by shopwareBot** [comment]
[public] Shopware automated response: This issue is linked to the internal issue: **NEXT-40922**



[created from NEXT-40922, comment 542986]
